Jenkinsfile

```diff
@@ -29,7 +29,7 @@ node {
           echo "Running a subset of the tests to check the content schema changes"
           govuk.runRakeTask("test:publishing_schemas --trace")
         } else {
-          govuk.runRakeTask("ci:setup:minitest test:in_parallel --trace")
+          govuk.runRakeTask("ci:setup:minitest test --trace")
```

Comment:
Could the commit message (on d57d170352115033e556b99a51dca5926b10cfb6) be updated to reflect that the problem has been fixed in a later commit, rather than it being fixed after the rails upgrade?